app/localization/src/DateFormatter.js

```diff
@@ -1,17 +1,61 @@
 // @flow strict
 
-import moment from 'moment/min/moment-with-locales'; // eslint-disable-line no-restricted-imports
-
 import getDeviceLocale from './GetDeviceLocale';
 
-// Language prop passed from native code is not accessible at this point.
-moment.locale(getDeviceLocale());
-
-// moment.forat('L') returns a string like DD/MM/YYYY or DD.MM.YYYY depending on current locale
-// This function will chop of the year
-moment.getLocalizedWithoutYear = (date: moment) => {
-  return moment(date)
-    .format('L')
-    .substring(0, 5);
-};
-export default moment;
+// language prop passed from native code is not accessible at this point
+const DEVICE_LOCALE = getDeviceLocale();
+
+function date(date: Date) {
+  return Intl.DateTimeFormat(DEVICE_LOCALE, {
+    timeZone: 'UTC', // this is very important!
+    weekday: 'short',
+    month: 'numeric',
+    day: 'numeric',
+  }).format(date);
+}
+
+function time(date: Date) {
+  return Intl.DateTimeFormat(DEVICE_LOCALE, {
+    timeZone: 'UTC', // this is very important!
+    hour: 'numeric',
+    minute: 'numeric',
+  }).format(date);
+}
+
+function pad(number) {
+  if (number < 10) {
+    return '0' + number;
+  }
+  return number;
+}
+
+/**
+ * Purpose of this wrapper is to make very opinionated and restrictive
+ * formatter. Every function is localized and you should never call formatting
+ * on the raw Date object.
+ *
+ * TODO: we should probably accept custom Value Object
+ */
+function DateFormatter(rawDate: Date) {
```

Comment:
could we do `rawDate: Date = new Date()` here as well?

Reply:
Why? There is not a single use-case for it and more importantly, it looks weird:

```
DateFormatter().formatToDate();
```

_"What date are you formatting here?"_ So, is it a good idea? I wanted to be explicit here...

Reply:
This is how moment works, you can do `moment().format('L')`

Reply:
That's not an argument for me... 😄

Reply:
Well, I will have to live with it then 🙈

Reply:
I will change it. As I said there is not a single use-case of `DateFormatter(new Date())` in our codebase... 🙂

Reply:
Ok, OCD calming.. 😄 Ping me when it is done

Reply:
Your OCD should be calmed down now. I also introduced new `formatToBirthday` in order to format birthdays. Not sure about the name but it's good enough at this point...